tpm2tools/misc_test.go

```diff
@@ -0,0 +1,43 @@
+package tpm2tools
+
+import (
+	"bytes"
+	"crypto/sha1"
+	"crypto/sha256"
+	"testing"
+
+	"github.com/google/go-tpm-tools/internal"
+	"github.com/google/go-tpm/tpm2"
+)
+
```

Comment:
We should also add a test that makes sure we get the right value after extending a PCR (this could just be a component of your table driven tests.

Reply:
Done. Only extended PCR0, but this should be fine. 

I'm tempted to extend the tests to compare the marshaled protos to strings, but I'm not sure it's worth how gross the table driven tests would need to look.

Reply:
No need to compare the marshaled strings, but compute the out values (like with `sha256.Sum(whatever)`).